### DISCUSSION TITLE
How to test what key is the MOD key?

### DISCUSSION THREAD

**Discussion by jidanni** [i3/i3#4714] [Q&A] [answered]
See below.

**Comment by jidanni** [comment]
Actually, it would be great if just holding down MOD would somehow make something in the corner light up.

**Comment by Airblader** [comment, marked as answer]
I wouldn't recommend choosing any configuration randomly and then trying to understand it, but rather understanding how to configure something and then configuring it. 

You can figure out what is set as the modifier key by looking at the configuration. You can figure out where the configuration lives by running `i3 --more-version`.

**Comment by jidanni** [comment]
Well it turned out I was watching a YouTube video about i3 in fullscreen mode.
It told me to do MOD+Enter to create shell windows.
So I did it many times... all underneath the screen. No wonder.

**Reply by Airblader** [reply]
If it helps, personally I have this script which I wrap my command to open terminals in:

```
#!/usr/bin/env bash
# vim:sw=4:ts=4:expandtab

# +++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
# + Usage: fullscreen_wrap.sh <command>
# +
# +   This script checks whether the focused window is currently in fullscreen
# +   mode. If so, it disables the fullscreen for it, runs the passed command
# +   and enables fullscreen for the window again once the command exits.
# +
# +   If the focused window is currently not in fullscreen mode, it will
# +   simply run the command and exit.
# +++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++

COMMAND="$@"
FOCUSED=$(xprop -root _NET_ACTIVE_WINDOW | awk -F' ' '{print $NF}')

if xprop -id ${FOCUSED} _NET_WM_STATE | grep -q _NET_WM_STATE_FULLSCREEN; then
    i3-msg "[id=${FOCUSED}] fullscreen disable"
    ${COMMAND}
    i3-msg "[id=${FOCUSED}] fullscreen enable"
else
    ${COMMAND}
fi
```
